app/models/document_type/primary_publishing_organisation_field.rb

```diff
@@ -0,0 +1,31 @@
+class DocumentType::PrimaryPublishingOrganisationField
+  def id
+    "primary_publishing_organisation"
+  end
+
+  def payload(edition)
+    { links: { primary_publishing_organisation: edition.tags["primary_publishing_organisation"] } }
```

Comment:
Minor: could we use the `id` here and similarly in other methods/models? We did this for [content fields](https://github.com/alphagov/content-publisher/blob/master/app/models/document_type/body_field.rb#L9), where the field is store in a JSON blob (so the naming is dictated by the code here), which applies to all tags.